testdata/go/memcached-operator/PROJECT

```diff
@@ -1,11 +1,14 @@
 domain: example.com
-layout: go.kubebuilder.io/v2
+layout: go.kubebuilder.io/v3-alpha
 projectName: memcached-operator
 repo: github.com/example/memcached-operator
 resources:
-- group: cache
+- crdVersion: v1
+  group: cache
   kind: Memcached
   version: v1alpha1
+  webhookVersion: v1
 version: 3-alpha
 plugins:
-  go.sdk.operatorframework.io/v2-alpha: {}
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {}
```

Comment:
Notice that these two plugins have been componentized from the go plugin wrapper. This was done to make their config objects more akin to how they'll look after they're implemented as phase 2 plugins. This doesn't break any projects because the old `go.sdk.operatorframework.io/v2-alpha` plugin still exists, but for `go.kubebuilder.io/v2` projects only.